bundesliga_app/admin.py

```diff
@@ -2,13 +2,8 @@
 from . import models
 
 
-class MemberTypeAdmin(admin.ModelAdmin):
-    list_display = ("type_number", "type_name")
-
-
 class DiscountAdmin(admin.ModelAdmin):
-    list_display = ("discount_name",)
```

Comment:
Sorry, one small advice in order to read code i'd rather have "discount_name" instead of "name" is more descriptive